specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,1178 @@
+        Long, // AppNotification stays on-screen for longer, and then goes into Notification Center.
+    };
+
+    enum ImageCrop
+    {
+        Default, // Uses the default renderer to display the image
+        Circle, // Crops the image as a circle.
+    }
+
+    runtimeclass AppNotificationContent
+    {
+        AppNotificationContent();
+
+        // Adds arguments to the launch attribute to return when AppNotification is clicked.
+        AppNotificationContent AddArgument(String key, String value);
+
+        // Sets the timeStamp of the AppNotification to when it was constructed instead of when it was sent.
+        AppNotificationContent SetTimeStamp(Windows.Foundation.DateTime value);
+
+        // Sets the scenario of the AppNotification.
+        AppNotificationContent SetScenario(AppNotificationScenario value);
+
+        // Adds text to the AppNotification.
+        AppNotificationContent AddText(String text);
+        AppNotificationContent AddText(String text, AppNotificationTextProperties properties);
+
+        AppNotificationContent SetAttributionText(String text);
+        AppNotificationContent SetAttributionText(String text, String language);
+
+        // Sets the full-width inline-image that appears when you expand the AppNotification
+        AppNotificationContent SetInlineImage(Windows.Foundation.Uri imageUri);
+        [default_overload]
+        AppNotificationContent SetInlineImage(Windows.Foundation.Uri imageUri, ImageCrop imageCrop);
+        AppNotificationContent SetInlineImage(Windows.Foundation.Uri imageUri, String alternateText);
+        AppNotificationContent SetInlineImage(Windows.Foundation.Uri imageUri, String alternateText, ImageCrop imagecrop);
+
+        // Sets the image that is left-aligned with notification text
+        AppNotificationContent SetAppLogoOverride(Windows.Foundation.Uri imageUri);
+        AppNotificationContent SetAppLogoOverride(Windows.Foundation.Uri imageUri, String alternateText);
+        AppNotificationContent SetAppLogoOverride(Windows.Foundation.Uri imageUri, ImageCrop imageCrop);
+        AppNotificationContent SetAppLogoOverride(Windows.Foundation.Uri imageUri, String alternateText, ImageCrop imageCrop);
+
+        // Sets the image that displays within the banner of the AppNotification.
+        AppNotificationContent SetHeroImage(Windows.Foundation.Uri imageUri);
+        AppNotificationContent SetHeroImage(Windows.Foundation.Uri imageUri, String alternateText);
+
+        // SetAudio
+        [default_overload]
+        AppNotificationContent SetAudio(Windows.Foundation.Uri audioUri);
+        AppNotificationContent SetAudio(AppNotficiationSoundEvent appNotficiationSoundEvent);
+
+        [default_overload]
+        AppNotificationContent SetAudio(Windows.Foundation.Uri audioUri, AppNotificationDuration duration);
+        AppNotificationContent SetAudio(AppNotficiationSoundEvent appNotficiationSoundEvent, AppNotificationDuration duration);
```

Comment:
type missing 'i' in appNot?ficiationSoundEvent

Reply:
and the i is in the wrong place twice :D
        AppNotificationContent SetAudio(AppNotificationSoundEvent appNotificationSoundEvent, AppNotificationDuration duration);